src/components/MaterialUI/DataDisplay/PredefinedElements/StepperModal.js

```diff
@@ -142,8 +142,13 @@ const StepperModal = ({
 	modalProps.set(ModalProps.propNames.backdropClickCallback, backdropCallback ?? closeCallback);
 	modalProps.set(ModalProps.propNames.type, type);
 
-	const nextClick = useCallback(() => changeCurrentStep(step => step + 1), []);
-	const backClick = useCallback(() => changeCurrentStep(step => step - 1), []);
+	const nextClick = useCallback(customHandler => {
+		if (customHandler) {
+			customHandler();
+		}
+		changeCurrentStep(step => step + 1);
+	}, []);
+	const backClick = useCallback(customHandle => changeCurrentStep(step => step - 1), []);
```

Comment:
customHandle (no R) ?

You can probably replace it by `()`

Reply:
Oh yeah, I changed my mind for the back click, forgetting that modif